app/celery/process_letter_client_response_tasks.py

```diff
@@ -12,10 +15,16 @@
     dao_update_notification,
 )
 from app.exceptions import NotificationTechnicalFailureException
+from app.models import LetterCostThreshold
 
 
 @notify_celery.task(bind=True, name="process-letter-callback")
-def process_letter_callback_data(self, notification_id, page_count, dvla_status, cost_threshold, despatch_date):
+def process_letter_callback_data(
+    self, notification_id: str, page_count: int, dvla_status: str, cost_threshold: str, despatch_date: date
+):
+    notification_id = uuid.UUID(notification_id)
```

Comment:
Very minor, but you could update the type hint for `dao_record_letter_despatched_on_by_id` now
https://github.com/alphagov/notifications-api/blob/cdf76491a01e8ef8c242e31a6ebf246a8447e9cc/app/dao/notifications_dao.py#L901

Reply:
done 👍